app/lnd/config/index.js

```diff
@@ -5,5 +5,5 @@
 export default {
   lightningRpc: `${__dirname}/rpc.proto`,
   lightningHost: 'localhost:10009',
-  cert: '/Users/jmow/Library/Application Support/Lnd/tls.cert'
+  cert: '/Users/joaoalmeida/Library/Application\ Support/Lnd/tls.cert'
```

Comment:
How should we handle this? Should remove anyones username probably

Reply:
`os.homedir()` https://nodejs.org/api/os.html#os_os_homedir  ?

Reply:
isn't os.userInfo().username better in this case ? @thisconnect

Reply:
@joaodealmeida `os.userInfo().username` works for me locally

Reply:
I got it working here as well, just wanted to make sure what was the correct function to use here. But I guess I'll go with userInfo().username since its working pretty well.

Reply:
cool